pkg/kev/services_test.go

```diff
@@ -27,12 +29,27 @@ var _ = Describe("ServiceConfig", func() {
 	Context("validation", func() {
 
 		When("base labels", func() {
+
+			// TODO: This needs a better test in a future iteration, the base labels are no longer just the required ones.
 			It("fails when base labels are not present", func() {
-				err := ServiceConfig{Labels: composego.Labels{}}.validate()
-				Expect(err).Should(MatchError(ContainSubstring(config.BaseServiceLabels[0])))
+				err := ServiceConfig{Labels: composego.Labels{
```

Comment:
It would benefit from splitting this block into separate cases as we expect a failure here (as title suggest), however, the last case validated the opposite.

Reply:
ah I see, good point

Reply:
done